spec/factories/courses.rb

```diff
@@ -47,6 +47,15 @@
     passcode 'pizza'
   end
 
+  factory :basic_course, class: 'BasicCourse' do
```

Comment:
I can just add this on after merging, if @AndrewGreen gives the +1 to this PR. https://github.com/WikiEducationFoundation/WikiEduDashboard/commit/4bff6f770b789f83f7ee31cf94d36e776b9d36f5